apps/estuary/lib/estuary/application.ex

```diff
@@ -12,12 +12,31 @@ defmodule Estuary.Application do
   def start(_type, _args) do
     validate_topic_exists()
 
-    children = []
+    children = [
+      {Elsa.Supervisor, elsa_options()}
+    ]
 
     opts = [strategy: :one_for_one, name: Estuary.Supervisor]
     Supervisor.start_link(children, opts)
   end
 
+  defp elsa_options do
+    [
+      endpoints: @elsa_endpoint,
```

Comment:
This is going to give you problems when you go to deploy.

Module attributes (`@thing`) are evaluated at compile time, so this is going to hardcode your Elsa endpoint rather than getting it from `runtime.exs`

Reply:
Agred